resources/config.rb

```diff
@@ -55,14 +55,14 @@
 
     tempfile = Tempfile.new(['hab_config', '.toml'])
     command = [ 'hab', 'config', 'apply', opts, new_resource.service_group,
-                incarnation, tempfile.path ]
+                incarnation, tempfile.path ].flatten.compact.join(' ')
     begin
       tempfile.write(TOML::Generator.new(new_resource.config).body)
       tempfile.close
       if Gem::Requirement.new('>= 14.3.20').satisfied_by?(Gem::Version.new(Chef::VERSION))
-        shell_out!(*command)
+        shell_out!(command)
```

Comment:
What's the rationale behind removing the splat operator here? Is it not needed anymore?

Reply:
the rationale is the same as the above issue for adding windows support.

`shell_out!` for windows doesn't support array args. We're joining the command array into one string command:

```
command = [ 'hab', 'config', 'apply', opts, new_resource.service_group,
                incarnation, tempfile.path ].flatten.compact.join(' ')
```